src/dimred/lsi/test.py

```diff
@@ -0,0 +1,206 @@
+import sys
+import pytest
+import subprocess
+import mudata as mu
+import numpy as np
+
+## VIASH START
+meta = {
+    'resources_dir': 'resources_test',
+    'executable': './target/docker/dimred/lsi/lsi',
+    'config': './src/dimred/lsi/config.vsh.yaml'
+}
+## VIASH END
+
+input_path = f"{meta['resources_dir']}/LSI/test_data.h5mu"
```

Comment:
@SarahOuologuem Can you reuse one of the pre-existing h5mu test resources or create a script in `resources_test_scripts` to generate this data?

Reply:
Yes, sorry!! Will do it next week!

Reply:
We need ATAC-specific test files for other components anyway. @rcannood or @DriesSchaumont , could you please run [this file](https://github.com/openpipelines-bio/openpipeline/blob/feature/ataq-demux/resources_test_scripts/cellranger_atac_tiny_bcl.sh) to download test files for ATAC? 

It will throw an error in the end (`cellranger_ataq_mkfastq` is not finished yet), but I need the motifs file to build the reference first. Then I can finish ATAC ingestion components, and we can use the same test files here

Reply:
@VladimirShitov I ran the script and uploaded the data to s3!